.github/workflows/publish-release.yml

```diff
@@ -19,11 +19,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Install JDK 11
+      - name: Install JDK 8
```

Comment:
This would mean we release conductor artifacts on JDK8 and it would readily be available on java 8 projects. Should not be a problem for java 8+ also.